x-pack/plugins/ingest_manager/common/types/rest_spec/agent_config.ts

```diff
@@ -11,7 +11,7 @@ export interface GetAgentConfigsRequest {
 }
 
 export interface GetAgentConfigsResponse {
-  items: AgentConfig[];
+  items: Array<AgentConfig & { agents?: number }>;
```

Comment:
If we all agree on this, then I can refactor some of the front end code to use this type (`GetAgentConfigResponse['items']`) instead of `AgentConfig` where needed (or maybe don't bother?)

Reply:
Should we have a type `AgentConfig` different for client side code that contains the agents count.

Reply:
I was wondering something similar when we discussed this in Slack. Naming is hard and I don't want to generalize too early. Let's defer until we need to use it again or have more examples.

Reply:
Sounds good. I'll hold off for now adjusting any front-end code.